code.go

```diff
@@ -40,10 +39,11 @@ var SourceCodeTypes = map[string]bool{
 }
 
 // parseCode parses a code address directive.
-func parseCode(addr []byte, file []byte) []byte {
+func (p *parser) parseCode(addr []byte, file []byte) []byte {
 	bytes.TrimSpace(addr)
 
-	textBytes, err := ioutil.ReadFile(string(file))
+	fullname := absname(p.cwd, string(file))
```

Comment:
It's not performance related, but more usage. I'm trying to keep everything []byte for as long as possible, only when std lib function require `string`, I make the conversion.
(there are probable a ton of places where this "rule" is broken though)